scripts/scripts_README.md

```diff
@@ -13,90 +13,99 @@ Most functions return a dictionary of lists (timeseries of a value)
 - line 47 : refueling length of reactor (int) -- input to ``import_data.write_reactors()``
-- line 58 : simulation duration (int) -- duration of simulation in months; dafault value of 
+- line 58 : simulation duration (int) -- duration of simulation in months; dafault value of
 780 months (65 years)
 - line 58 : burnup of reactors in CYCLUS simulation (int) -- must be a value in ``burnups``,
 so that recipe files are present; default of 50 GWd/MTU
 
 
-Outputs : 
-- ``database/reactors_pris_####.csv`` : ``.csv`` file of condensed PRIS year-end Reactor 
+Outputs :
+- ``database/reactors_pris_####.csv`` : ``.csv`` file of condensed PRIS year-end Reactor
 Status information, in which ``####`` is the four digit year the data is pulled from
-- ``input/project/inputs/region.xml`` : CYCLUS input file, in which ``project`` and ``region`` 
+- ``input/project/inputs/region.xml`` : CYCLUS input file, in which ``project`` and ``region``
 are user specifications
-- ``input/project/inputs/region/recipes/`` : directory containing recipe files that are read into 
+- ``input/project/inputs/region/recipes/`` : directory containing recipe files that are read into
 the CYCLUS input file, in which ``project`` and ``region`` are user specifications
-- ``input/project/inputs/region/reactors'`` : directory containing input files for each of the 
-reactors in the selected region. Each reactor will be a separate ``.xml`` input file that 
-will be read into the CYCLUS input file, in which ``project`` and ``region`` are user 
+- ``input/project/inputs/region/reactors'`` : directory containing input files for each of the
+reactors in the selected region. Each reactor will be a separate ``.xml`` input file that
+will be read into the CYCLUS input file, in which ``project`` and ``region`` are user
 specifications
 - ``input/project/inputs/region/buildtimes/inclusions.xml`` : Input file that contains links
-to the reactors in the region to be included in the CYLCUS simulation, in which ``project`` 
+to the reactors in the region to be included in the CYLCUS simulation, in which ``project``
 and ``region`` are user specifications
-- ``input/project/inputs/region/buildtimes/country/deployinst.xml`` : Input file of the 
-``DeployInst`` institution for the specified ``country`` that is read into the CYCLUS 
+- ``input/project/inputs/region/buildtimes/country/deployinst.xml`` : Input file of the
+``DeployInst`` institution for the specified ``country`` that is read into the CYCLUS
 input file, in which ``project`` and ``region`` are user specifications
 
 To run:
 ```
 python create_input.py
 ```
 
+### reactor_deployment.py
+This script holds schemes to deploy reactors to meet a need over time. They do
+not create cyclus input files, the outputs are dataframes. This script is not
+intended to be run as a whole, the individual functions are meant to be called
+in your analysis.
+
+Usage:
+    import reactor_deployment as dep
```

Comment:
If you change it to a class `from reactor_deployment import CLASSNAME as dep` maybe?